resources/public/api/conf/1.0/schemas/list_charges_response.json

```diff
@@ -0,0 +1,124 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "List self assessment charges",
+  "description": "List self assessment charges",
+  "type": "object",
+  "properties": {
+    "charges": {
+      "description": "A list of charges.",
+      "type": "array",
+      "items": {
+        "title": "charge",
+        "type": "object",
+        "properties": {
+          "taxYear": {
+            "description": "The tax year to which the charge relates.",
+            "type": "string",
+            "pattern": "^\\d{4}-\\d{2}$",
```

Comment:
I don't know if the date fields should have a regex. I think adding 'Date in the format: YYYY-YY' might be clearer.